index.js

```diff
@@ -1 +1,43 @@
 // Write your solution here!
+
+const cats = ["Milo", "Otis", "Garfield"];
+
+function destructivelyAppendCat(name){
+
+    return cats.push(name);
+}
+
+function destructivelyPrependCat(name){
+
+    return cats.unshift(name);
+}
+
+function destructivelyRemoveLastCat(){
+
+    return cats.pop();
+}
+
+function destructivelyRemoveFirstCat(){
+
+    return cats.shift();
+}
+
+function appendCat(name){
+    let updatedCats = [];
+    return updatedCats = [...cats, name];
```

Comment:
You could simplify this and the following by doing: 
```suggestion
    return [...cats, name];
```